test/simulator/timestamp/src/include/session_simulator.h

```diff
@@ -85,6 +85,7 @@ class session_simulator {
     bool _ts_round_read;
     bool _txn_running;
     bool _prepared_txn;
+    bool _durable_ts_set;
```

Comment:
We have `has_commit_ts` and `..._set`, we could be consistent here too.

Reply:
The reason I used set instead of has for durable ts is that we can have a durable ts (which is equal to the commit ts) or we can explicitly set the durable ts (by passing durable ts).